packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/AddCard/AddCard.tsx

```diff
@@ -160,93 +136,72 @@ const AddCardModal = ({ isModalOpen, onClose }: IAddCardModalProps) => {
   }
 
   return (
-    <CustomModal
-      className={classes.modalRoot}
-      injectedClass={{
-        inner: classes.modalInner
-      }}
-      active={isModalOpen}
-      closePosition="none"
-      maxWidth="sm"
-    >
-      <form onSubmit={handleSubmitPaymentMethod}>
-        <div
-          className={classes.root}
-          data-cy="modal-add-card"
-        >
-          <Grid
-            item
-            xs={12}
-            sm={12}
-          >
-            <Typography
-              className={classes.heading}
-              variant="h4"
-              component="h4"
-            >
-              <Trans>Add a credit card</Trans>
-            </Typography>
-          </Grid>
-          <CardNumberElement
+    <form onSubmit={handleSubmitPaymentMethod}>
+      <div
+        className={classes.root}
+        data-cy="modal-add-card"
```

Comment:
That's not a modal any more. 
```suggestion
        data-cy="form-add-card"
```